spec/serializers/api/v3/serializable_course_spec.rb

```diff
@@ -0,0 +1,216 @@
+require "rails_helper"
+
+describe API::V3::SerializableCourse do
+  let(:jsonapi_renderer) { JSONAPI::Serializable::Renderer.new }
+  let(:draft_enrichment) { build :course_enrichment }
+  let(:published_enrichment) { build :course_enrichment, :published }
+  let(:date_today) { Time.zone.today }
+  let(:time_now) { Time.now.utc }
+  let(:course) do
+    create(:course, enrichments: [draft_enrichment, published_enrichment], start_date: time_now, applications_open_from: date_today, level: :primary)
+  end
+  let(:course_json) do
+    jsonapi_renderer.render(
+      course,
+      class: {
+        Course: API::V3::SerializableCourse,
+      },
+    ).to_json
+  end
+  let(:parsed_json) { JSON.parse(course_json) }
+
+  subject { parsed_json["data"] }
+
+  it { should have_type("courses") }
```

Comment:
I think `should` is on it's way to being deprecated if it isn't already. `it { is_expected.to` is the new hotness.

Reply:
Ya. mea culpa, I'm just attached to `should`, but we'll probably have to do a bulk switch over to `is_expected`.

Reply:
NIce tip @gpeng. I'll change it here but as @misaka says we should do the entire code base  at some stage